### DISCUSSION TITLE
Minio peer expansion error, indicating incorrect file content

### DISCUSSION THREAD

**Discussion by smallemployees** [minio/minio#20698] [Q&A]
## NOTE
If this case is urgent, please subscribe to [Subnet](https://min.io/pricing) so that our 24/7 support team may help you faster.

## Expected Behavior

How can I implement the steps for peer-to-peer scaling and define a new pool

## Current Behavior

Unable to initialize backend: /data/minio/data drive is already being used in another erasure deployment. (Number of drives specified: 8 but the number of drives found in the 1st drive's format.json

## Steps to Reproduce (for bugs)

none
1.
2.
3.
4.

## Your Environment

* Version used (`minio --version`): RELEASE.2023-03-22T06-36-24
* Server setup and configuration:
* Operating System and version (`uname -a`):

**Comment by marktheunissen** [comment]
hi @smallemployees please follow our documentation here: https://min.io/docs/minio/linux/operations/install-deploy-manage/expand-minio-deployment.html

**Comment by marktheunissen** [comment]
> Unable to initialize backend: /data/minio/data drive is already being used in another erasure deployment. (Number of drives specified: 8 but the number of drives found in the 1st drive's format.json

This error indicates you are trying to use an existing MinIO in a new topology, e.g. you have 4 nodes, now you're trying to use those same 4 nodes in a 6 node setup. MinIO does not support this, you must add a new pool
